zookeeper-server/src/main/java/org/apache/zookeeper/server/backup/BackupConfig.java

```diff
@@ -18,28 +18,100 @@
 
 package org.apache.zookeeper.server.backup;
 
+import java.io.File;
+import java.util.Optional;
+import java.util.Properties;
+
+import org.apache.zookeeper.common.ConfigException;
+import org.apache.zookeeper.server.util.VerifyingFileFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 /**
- * Backup-related configurations
+ * Backup-related configurations.
  */
 public class BackupConfig {
-  public static final String NFS_BACKUP_STORAGE = "NFSBackupStorage";
+  private static final Logger LOG = LoggerFactory.getLogger(BackupConfig.class);
+
+  /*
+   * Default constants
+   */
+  private static final boolean DEFAULT_ENABLED = false;
```

Comment:
NIT: let's have the name more detailed such as DEFAULT_BACKUP_CONFIG_ENABLED

Reply:
I can rename it to `DEFAULT_BACKUP_ENABLED`. Backup config enabled isn't strictly correct.

Reply:
Yeah. Rename to DEFAULT_BACKUP_ENABLED should be fine. DEFAULT_ENABLED is a little bit confusing.